### ISSUE TITLE
Refactor of toggle buttons

### ISSUE THREAD

**Issue by splendido** [MeteorPackaging/autopublish.meteor.com#20]
I'm thinking to start a refactor of the code to end up showing three different toggle buttons.
1. enable auto-publish (for repositories containing a Meteor package)
2. simple webhook creation (for 3rd party library we're going to wrap)
3. enable auto-publish for repo with wrapped packages (which is: repository being under MeteorPackages and including a `autopublish.json` file..)

**Comment by dandv** [comment]
Button #1 seems like an easy convenience, right? It would only save authors from running `meteor publish` after `git push`, if I understand correctly.

For #3, did you mean "repo with wrapped package_s_", i.e. multiple packages?

**Comment by splendido** [comment]
yeah, n.1 would be for private packages (would we really like to support this?)
n.3 would be for our repositories wrapping 3rd party libraries like [snapsvg-wrapper](https://github.com/MeteorPackaging/snapsvg-wrapper) to cite the latest one...

**Comment by zimme** [comment]
I use autopublish for all my packages, but I guess I could do manual `meteor publish` whenever I release

**Comment by splendido** [comment]
@zimme it's fine you use autopublish!
...you're giving a bunch of valuable feedbacks this way, so please do use it.

The question:

> would we really like to support this?

is more for the long term, just in case too many people started to use it for private packages.

I already have a list of hooks with their own approval status, but I'm actually ignoring it when receiving triggers...
We might think to really consider approvals... but this would mean having someone taking care of new subscription requests, which means time...
...perhaps in case MDG will show some more interest on this ;-)

Btw, I've just turned users @zimme and @dandv  `admin` users so you can view some more button... :-)

**Comment by zimme** [comment]
:+1:
